src/ophyd_async/core/device_save_loader.py

```diff
@@ -59,6 +76,10 @@ def get_signal_values(
         key: signal for key, signal in signals.items() if key not in ignore
     }
     selected_values = yield Msg("locate", *selected_signals.values())
+
+    if isinstance(selected_values, dict):
+        selected_values = [selected_values]
+
```

Comment:
I want to know people's opinion on this; while writing tests I found that, if get_signal_values is called using a dictionary containing just one device, the result of yield Msg("locate", ...) will be a dictionary, not a sequence. The type hinting for selected_values suggests it will always be a sequence. But it isn't... Personally I'd prefer if locate always returned a list of dicts, instead of sometimes returning a dict and sometimes returning a list of dicts.

Thoughts?

Reply:
@rosesyrett Since we are merging this for need then coming back to it, can I just put a type ignore on this to get mypy to behave? It's the only thing flagging now

Reply:
Yes please. The type hints are wrong; this is probably a bluesky bug.